RobotCode/snobot2018/src/org/snobot/positioner/Positioner.java

```diff
@@ -129,7 +129,6 @@ public double getTotalDistance()
     @Override
     public void setPosition(double aX, double aY, double aAngle)
     {
-        mDriveTrain.resetEncoders();
```

Comment:
This was called twice.  Probably didn't hurt anything, but got rid of the duplication